packages/account/src/providers/fuel-core-schema.graphql

```diff
@@ -387,7 +389,6 @@ type GasCosts {
   divi: U64!
   ecr1: U64!
   eck1: U64!
-  ed19: U64!
```

Comment:
Should this be updated in the [chainConfig](https://github.com/FuelLabs/fuels-ts/blob/fd95a20d87038b30aa1d03fc356648027606ff57/.fuel-core/configs/chainConfig.json#L60)?

Reply:
This field has changed a few times since Thursday last week, it's now settled on being a dependent cost field.